PopulateDatabase.py

```diff
@@ -106,6 +108,8 @@ def save_question(question_name=None, question_text=None, choices=None, correct_
     question.answers.add(*answers)
     question.save()
 
+# Build all the groups and permissions
```

Comment:
This comment is redundant. If you don't think it is you should consider changing the method name.